source/documentation/deploying_apps/stopping_and_deleting_apps.md

```diff
@@ -40,13 +40,30 @@ If you accidentally delete your app without the ``-r`` option, you can delete th
 
 ``cf delete-route [domain name] --hostname [hostname]``
 
-### Deleting services
-Before deleting your app, you will need to delete any services you have provisioned for it.
+### Delete a service instance
 
-You can check the details of these by using the ``cf services`` command to see all active services in the current space. You will be able to see which services are bound *only* to your app - don’t delete services that other apps are also using.
+You must unbind a service instance from any apps it is bound to before you can delete that service instance. Note:
 
-You can then use this information to run
+- you do not need to delete an app's service instances before you can delete that app
+- a service instance can be bound to more than one app 
 
-``cf delete-service [SERVICE INSTANCE]``
+1. Run `cf services` to see all service instances in your targeted space. You will see the output as per this example:
 
-When all services are removed you can then delete your app, as above.
+    ```
+    name      service     plan    bound apps    last operation
+    mystuff   service1    small   my-app        create succeeded
```

Comment:
Maybe we can make `service{1,2}` concrete examples of things we have on the platform such as `postgres` and `mysql`?